it/utils/JourneyDataMongoHelper.scala

```diff
@@ -20,24 +20,24 @@ import org.scalatest.{BeforeAndAfterEach, Suite}
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.libs.json.{JsObject, Json}
 import play.api.test.Helpers._
-import reactivemongo.play.json.ImplicitBSONHandlers.JsObjectDocumentWriter
 import uk.gov.hmrc.soletraderidentification.repositories.JourneyDataRepository
 
-import scala.concurrent.ExecutionContext.Implicits.global
 
 trait JourneyDataMongoHelper extends BeforeAndAfterEach {
   self: GuiceOneServerPerSuite with Suite =>
 
   lazy val repo: JourneyDataRepository = app.injector.instanceOf[JourneyDataRepository]
 
-  def findById(journeyId: String): Option[JsObject] =
-    await(repo.collection.find(
-      Json.obj("_id" -> journeyId),
-      None
-    ).one[JsObject])
+  def findById(journeyId: String, authInternalId: String): Option[JsObject] =
+    await(repo.getJourneyData(journeyId, authInternalId))
 
   def insertById(journeyId: String, internalId: String, jsonData: JsObject = Json.obj()): Unit =
-    await(repo.collection.insert(true).one(Json.obj("_id" -> journeyId, "authInternalId" -> internalId) ++ jsonData))
+    await(repo.collection.insertOne(
+      Json.obj(
+        "_id" -> journeyId,
+        "authInternalId" -> internalId
+      ) ++ jsonData
+    ).toFuture())
```

Comment:
The method insertOne returns an instance of SingleObservable. To really match the declared return type you could use ".toFuture().map(_ => Unit)".

Reply:
done